scripts/directives/cypress-stub-example.js

```diff
@@ -0,0 +1,30 @@
+function processNode(node, { _require, error }) {
+  const helpers = _require(__dirname, './helpers/example-helpers')
+  const { children } = helpers.getNodeProperties(node)
+  const { errorArgs, parts } = helpers.getCodeBlocks(children, { count: 2 })
+
+  if (errorArgs) {
+    return error(...errorArgs)
+  }
+
+  const [e2eExample, componentExample] = parts
+
+  return helpers.getCodeGroup(
+    {
+      label: 'End-to-End Test',
+      language: 'js',
+      body: e2eExample,
+    },
+    {
+      label: 'Component Test',
+      language: 'js',
+      body: componentExample,
+    }
+  )
+}
+
+module.exports = {
+  type: 'containerDirective',
+  name: 'cypress-stub-example',
```

Comment:
I referred to it as the stub example because `cy.stub()` is used in both docs refs. And in the CT example there is no `onBeforeLoad` hook.

Reply:
It seems like this should just be called `cypress-e2e-component-example`.